### DISCUSSION TITLE
I want to add the TAB key to KeyboardAccelerators.

### DISCUSSION THREAD

**Discussion by dimohy** [microsoft/WindowsAppSDK#1800] [Q&A] [answered]
In WinUI 3,
I want to add the TAB key to 'KeyboardAccelerators', but it doesn't seem to work because of the behavior of TabNavigation. Is there any way to make this possible?

**Comment by andrewleader** [comment, marked as answer]
Hi @dimohy, since this is a WinUI 3 question, you can ask it in the WinUI repo: https://github.com/microsoft/microsoft-ui-xaml/issues
